### PR TITLE
Add IPv6 listen and clients for ABFAB

### PR DESCRIPTION
**Pull request by alejandro-perez**

### CODE REVIEW

File: raddb/sites-available/abfab-tls [hunk 1]
```diff
@@ -40,6 +40,43 @@ listen {
 	clients = radsec-abfab
 }
 
+# There needs to be a separated "listen" section for IPv6.
+# Typically it will be identical to the IPv4 one above, but there might be
+# some differences (e.g. if a different certificate or port is desired)
+listen {
+	ipaddr = ::
```
Comment (jpereira) [inline raddb/sites-available/abfab-tls]: it should be: `ipv6addr`

File: raddb/sites-available/abfab-tls [hunk 2]
```diff
@@ -52,6 +89,11 @@ clients radsec-abfab {
 		proto = tls
 	}
 
+        client default_ip6 {
+		ipaddr = ::/0
```
Comment (jpereira) [inline raddb/sites-available/abfab-tls]: `ipv6addr` instead of `ipaddr`

right @alandekok ?

File: raddb/sites-available/abfab-tls (hunk 1 above)
Comment (alejandro-perez) [reply raddb/sites-available/abfab-tls]: Well, according to the documentation, ipaddr works (indeed, I've tested this configuration). A different thing is whether using `ipv6addr` is preferred. In that case I guess a similar case for `ipv4addr` is recommended

File: raddb/sites-available/abfab-tls (hunk 2 above)
Comment (arr2036) [reply raddb/sites-available/abfab-tls]: No, ipaddr works.  It figures out the type of IP address from the formatting.

The only time you really need to use ipv4addr or ipv6addr is when you want to control whether the A or AAAA is used as the result of DNS resolution.

**Pull request merged**